app/views/denounces/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="row justify-content-center mt-5">
+  <div class="col-xs-12 col-xl-10">
+    <div class="card mb-3 shadow">
+      <div class="card-header">
+        <h3>Edit Denounce</h3>
+      </div>
+      <%= render 'form', denounce: @denounce %>
+      <!-- <%= link_to 'Show', @denounce, class: "btn btn-primary mx-auto" %> -->
```

Comment:
I prefer to remove if don't plan to use

Reply:
It is indeed being use in the My denounces view.